oshi-core/src/main/java/oshi/hardware/HardwareAbstractionLayer.java

```diff
@@ -77,13 +77,22 @@
     List<HWDiskStore> getDiskStores();
 
     /**
-     * Gets a list of {@link NetworkIF} objects, representing a network interface
+     * Gets a list of inet {@link NetworkIF} objects, representing a network interface
```

Comment:
I'd prefer not to add `inet` here as it's not clearly defined, and the object returned is part of the OSHI API.

I see throughout the changes you've used this term, though, so perhaps you need to just clearly define what "inet" means.  Or use a more verbose term that's more clear.